Base/usr/share/man/man1/test.md

```diff
@@ -0,0 +1,92 @@
+## Name
+
+test - checks files and compare values
+
+## Synopsis
+
+```**sh
+$ test [expression]
+$ [ expression ]
```

Comment:
```suggestion
$ [ [expression] ]
```

Reply:
that's _so_ confusing though...
I'll add `[ ]` instead.

Reply:
Yeah that is becoming confusing, maybe let's just drop the inner brackets altogether for this synopsis?

Reply:
Then I'll make it also say `$ test`, for completeness.

Reply:
> Then I'll make it also say `$ test`, for completeness.

You should also add `$ [ ]` then.